Alert/Alert.css

```diff
@@ -0,0 +1,134 @@
+@import "../../utility/component.css";
+
+.alert{
+    position: relative;
+    padding: 1rem 1rem;
+    margin-bottom: 1rem;
+    border: 1px solid transparent;
+    border-radius: 0.25rem;
+}
+
+/*---alert example---*/
+
+#show_alert{
+    display: none;
+}
+
+#alert_example{
+    display: none;
+    position: absolute;
+    z-index: 1;
+    right: 0;
+    z-index: 999;
+    transform: translateX(100%);
+    -webkit-transform: translateX(100%);
+    animation: slide-in 0.5s forwards;
+    -webkit-animation: slide-in 0.5s forwards;
+}
+
+/*----alert-close-icon---*/
+
+.close_alert{
+  margin-left: 0.938rem;
+  color: white;
+  font-weight: bold;
+  float: right;
+  font-size: 1.375rem;
+  line-height: 1.25rem;
+  cursor: pointer;
+  z-index: 1;
+  transition: 0.3s;
+}
+
+/*----action-alert---*/
+
+.action_alert{
+  display: block;
+  display: flex;
+  justify-content: space-between;
+}
+
+  @keyframes slide-in {
+    0% {
+      -webkit-transform: translateX(100%);
+    }
+    100% {
+      -webkit-transform: translateX(0%);
+    }
+  }
+  
+  @-webkit-keyframes slide-in {
+    0% {
+      transform: translateX(100%);
+    }
+    100% {
+      transform: translateX(0%);
+    }
+  }
+  
+
+/*--------outline---border----*/
+
+
+.alert-outline-success{
+    border : 1px solid var(--green);
+}
+
+.alert-outline-info{
+    border : 1px solid var(--blue);
+}
+
+.alert-outline-warning{
+    border : 1px solid var(--yellow);
+}
+
+.alert-outline-danger{
+    border : 1px solid var(--red);
+}
+
+.alert-outline-default{
+    border : 1px solid var(--black);
+}
+
+/*--------fill---alert*/
+.alert-fill-box-success{
+  color:var(--green);
+  background-color: var(--background-color-green);
```

Comment:
Same as mentioned above, better variable names and ad short as possible, like for this instance you can use var(--alert-success-bg)

Reply:
ok